### PR TITLE
Assignment5 @shuywalker @linhtran8699

### PR DESCRIPTION
**Pull request by tremorrisdev**
https://polite-desert-067c8041e.1.azurestaticapps.net/

### CODE REVIEW

**Comment by dcox2005** [comment]
where is your front end site and you api sites?

**Comment by hewrtarkhany** [comment]
great job it shows team work

File: wordle-web/components/searchbar.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
```
Comment (DakotaCondos) [inline wordle-web/components/searchbar.vue:1]: I like that you are doing this as a component

**Review by DakotaCondos — COMMENTED** [review]
It looks like the identity stuff isnt in this pull request yet

**Comment by mmwoodfo** [comment]
@schuywalker Did you do a PR review? I can't seem to find one.

**Comment by mmwoodfo** [comment]
@schuywalker Did you do a PR review? I can't seem to find one.

**Pull request closed without merging**